.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -37,6 +43,16 @@ A clear and concise description of any alternative solutions or features you've
 Add any other information that may help people understand your I2I.
 -->
 
+## Technical Reviews
+<!--
+This sections links to the relevant reviews that the launch of this change is gated on. 
+If one of these reviews is not needed, please state the reasoning here instead.
+If linking to documents please make sure that they are accessible by anyone.
+-->
+- Accessibility review
```

Comment:
Lastly, here too, it would be good to offer structure and example of each of those review types.

Reply:
Taken a stab at this inline. Don't have examples for these.

Reply:
Well duh! Could we work on a template for them? Could they even be inline? How does Chromium do it, for example?

Reply:
Inlined here?

Reply:
Sorry, I wasn't very clear. Let's discuss that in #24132 too.